### PR TITLE
Add ignore-missing-samples option.

### PR DESCRIPTION
**Pull request by ShifaSZ**
Add an ignore-missing-samples option to allow testing on partial mitochondrial callset.

### CODE REVIEW

File: luigi_pipeline/lib/hail_tasks.py [hunk 1]
```diff
@@ -123,8 +124,7 @@ def run_vep(mt, genome_version, runner='VEP', vep_config_json_path=None):
 
         return runners[runner]().run(mt, genome_version, vep_config_json_path=vep_config_json_path)
 
-    @staticmethod
-    def subset_samples_and_variants(mt, subset_path):
+    def subset_samples_and_variants(self, mt, subset_path):
```
Comment (ShifaSZ) [inline luigi_pipeline/lib/hail_tasks.py:127]: Change the method type to access the `ignore_missing_samples` parameter.

**Comment by hanars** [comment]
We only want this to be an option for the mitpo loading so we should not be changing the base class at all

**Comment by ShifaSZ** [comment]
> We only want this to be an option for the mito loading so we should not be changing the base class at all

I could change the new Luigi parameter `ignore-missing-sample` to the mito class. But I have to change the base class method to use the parameter. Or should I overwrite the method in the mito class?

**Comment by hanars** [comment]
override the method in the mito class

**Pull request closed without merging**